.github/workflows/ci.yml

```diff
@@ -22,6 +22,9 @@ jobs:
   build-rti:
     uses: lf-lang/reactor-c/.github/workflows/build-rti.yml@main
 
+  build-trace-tools:
+    uses: lf-lang/reactor-c/.github/workflows/build-trace-tools.yml@move-tracing-utils
```

Comment:
Should this be `@main`?

Reply:
Yes. I will rename to main. But just want to run the CI pipeline before merging